projects/cups/project.yaml

```diff
@@ -1,14 +1,27 @@
 homepage: "https://openprinting.github.io/cups/"
+main_repo: 'https://github.com/OpenPrinting/cups'
+# help_url:
 language: c
-primary_contact: "security@msweet.org"
+
+primary_contact: "jiongchiyu@gmail.com"
 auto_ccs:
-  - "ajsinghyadav00@gmail.com"
```

Comment:
Should we be taking @pkillarjun off?

Reply:
> Should we be taking @pkillarjun off?
Yes.

I think it would be better if one of the maintainers of the cups approved these changes, isn't it?

Reply:
> Should we be taking @pkillarjun off?

During our group meeting last week, we suppose exposing the unfixed bugs to members outside the OpenPrinting security team could potentially pose security risks and would bring maintainance inconvenience. Therefore, though we greatly appreciate Arjun's contribution, we have currently removed him from the mailing list. @tillkamppeter @iosifache

Reply:
> we suppose exposing the unfixed bugs to members outside the OpenPrinting security team could potentially pose security risks and would bring maintainance inconvenience.

I don't fix bugs. The reason for that is pretty clear, I believe.
Also, I only tried to fix the fuzz blocker and the bug in my harness code: https://github.com/lldpd/lldpd/pull/602, https://github.com/lldpd/lldpd/pull/596, https://github.com/lldpd/lldpd/pull/587. For that, I need my fuzzers to be in the upstream repo.